Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/ActorData.xml

```diff
@@ -29853,13 +29862,13 @@
         <!-- Note: Don't swap the model at actor creation, as that overrides the warp-in model -->
         <On Terms="UnitBirth; ValidatePlayer AP_HaveZealotWhirlwind" Send="ModelSwap AP_ZealotAiur"/>
         <On Terms="Upgrade.AP_VoidZealotWhirlwind.Add" Send="ModelSwap AP_ZealotAiur"/>
-        <On Terms="UnitDeathCustomize; ValidatePlayer AP_HaveZealotWhirlwind" Send="DeathCustomize AiurZealot"/>
+        <On Terms="UnitDeathCustomize; ValidatePlayer AP_HaveZealotWhirlwind" Send="DeathCustomize Variation0"/>
         <Model value="Zealot"/>
         <ModelFlags index="OutlineEmitter" value="1"/>
         <BuildModel value="ZealotWarpIn"/>
         <DeathArray index="Normal" ModelLink="ZealotDeath" SoundLink="Zealot_DeathFX"/>
         <DeathCustoms ModelLink="HallucinationDeath" SoundLink="Sentry_HallucinationDeathSmall" Name="Hallucination"/>
-        <DeathCustoms ModelLink="AP_ZealotAiurDeath" SoundLink="Zealot_DeathFX" Name="AiurZealot"/>
+        <DeathCustoms ModelLink="AP_ZealotAiurDeath" SoundLink="Zealot_DeathFX" Name="Variation0"/>

```

Comment:
This change shouldn't have been there IIRC

Reply:
It should be -- I discovered while working on the Colossus death that only certain strings are allowed to name a DeathCustom. `AiurZealot` is not on it. So this fixes something that was broken, it just wasn't easy to see on zealots because their death animations are so similar.

The complete list of allowed names is in core.sc2mod's actordata, in DeathCustomPriorityList.

Reply:
This is already patched by allowing `AiurZealot` there at the top of the actor file